priv/templates/extended_bin

```diff
@@ -343,6 +343,24 @@ case "$1" in
              "unpack" "$REL_NAME" "$NAME_TYPE" "$NAME" "$COOKIE" "$2"
         ;;
 
+    uninstall)
```

Comment:
I've done a bit of refactoring in #505 (input on this PR is welcome btw), maybe we can rebase this PR onto master when it is merged